### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement pycolmap 

### ISSUE THREAD

**Issue by iqfareez** [colmap/pycolmap#134]
My `requirements.txt` file:
```
pycolmap
```

i tried
```
> pip install -r .\requirements.txt 
```

But got error:
```
ERROR: Could not find a version that satisfies the requirement pycolmap (from versions: none)
ERROR: No matching distribution found for pycolmap
```

Any idea how to solve this issue? Thanks

Python version: Python 3.9.13

**Comment by Skydes** [comment]
This is odd, we already have wheels for 3.9, with both manylinux and macos. What's your platform/OS? Can you try again with the new release that just got published?

**Comment by iqfareez** [comment]
Sorry.

Recently, Google Colab [upgraded](https://medium.com/google-colab/colab-updated-to-python-3-10-27eb02daa162) their default runtime to python 3.10. It had me confused. I can run if I use the fallback version.

But, since you already published the 3.10 version, I can now run again on colab without problems. https://github.com/cvg/Hierarchical-Localization/issues/271#issuecomment-1540834159

I'll now close this
